andela/travel_tool_back
# #162813634 Verified trips should be included in the guesthouse details

#### What does this PR do?
- Verified trips should be included in the guesthouse details

#### Description of Task to be completed?
Verified trips should be included in the guesthouse details

#### How should this be manually tested?
* Clone the repo with ```git clone https://github.com/andela/travel_tool_back.git``` .
* cd to ```travel_tool_back``` 
* Checkout to the branch ```bg-fix-guesthouse-details-bug-162813634 ```.
* Start up both the back-end and the front-end server with ```yarn start:dev``` and  ```yarn start``` respectively.
* Make requests, approve them, fill the checklists, and verify them.
* Navigate to ```Manage``` on the ```Residence``` menu.
* Select the Guesthouse for requester's choice 
* There should be details for both approved and verified requests

#### Any background context you want to provide?
You need the role of ```Travel admin``` to view the guesthouse details.

#### What are the relevant pivotal tracker stories?
[162813634](https://www.pivotaltracker.com/story/show/162813634)

#### Screenshots (if appropriate)

#### Questions:
